File: Examples/CSharp/PivotTableExamples/FormattingLook.cs
using System.IO;

using Aspose.Cells;
using System.Drawing;
using Aspose.Cells.Pivot;

namespace Aspose.Cells.Examples.PivotTableExamples
{
    public class FormattingLook
    {
        public static void Main(string[] args)
        {
            //ExStart:1
            // The path to the documents directory.
            string dataDir = Aspose.Cells.Examples.Utils.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType);

            //Load a template file
            Workbook workbook = new Workbook(dataDir + "Book1.xls");

            //Get the first worksheet
            Worksheet worksheet = workbook.Worksheets[0];
            var pivot = workbook.Worksheets[0].PivotTables[0];

            pivot.PivotTableStyleType = PivotTableStyleType.PivotTableStyleDark1;

            Style style = workbook.CreateStyle();
            style.Font.Name = "Arial Black";
            style.ForegroundColor = Color.Yellow;
            style.Pattern = BackgroundType.Solid;

            pivot.FormatAll(style);

            //Saving the Excel file
            workbook.Save(dataDir + "output.xls");

            //ExEnd:1

        }
    }
}

File: Examples/VisualBasic/PivotTable/FormattingLook.vb
Option Infer On

Imports System.IO

Imports Aspose.Cells
Imports System.Drawing
Imports Aspose.Cells.Pivot

Namespace PivotTableExamples
    Public Class FormattingLook
        Public Shared Sub Run()
            ' ExStart:1
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType)

            ' Load a template file
            Dim workbook As New Workbook(dataDir & "Book1.xls")

            ' Get the first worksheet
            Dim worksheet As Worksheet = workbook.Worksheets(0)
            Dim pivot = workbook.Worksheets(0).PivotTables(0)

            pivot.PivotTableStyleType = PivotTableStyleType.PivotTableStyleDark1

            Dim style As Style = workbook.CreateStyle()
            style.Font.Name = "Arial Black"
            style.ForegroundColor = Color.Yellow
            style.Pattern = BackgroundType.Solid

            pivot.FormatAll(style)

            ' Saving the Excel file
            workbook.Save(dataDir & "output.xls")

            ' ExEnd:1

        End Sub
    End Class
End Namespace
